app/Http/Controllers/ItemController.php

```diff
@@ -369,26 +369,6 @@ public function appload(Request $request): ?string
         if ($appid === 'null') {
             return null;
         }
-        /*$appname = $request->input('app');
-        //die($appname);
-
-        $app_details = Application::where('name', $appname)->firstOrFail();
-        $appclass = $app_details->class();
-        $app = new $appclass;
-
-        // basic details
-        $output['icon'] = $app_details->icon();
-        $output['name'] = $app_details->name;
-        $output['iconview'] = $app_details->iconView();
-        $output['colour'] = $app_details->defaultColour();
-        $output['class'] = $appclass;
-
-        // live details
-        if($app instanceof \App\EnhancedApps) {
-            $output['config'] = className($app_details->name).'.config';
-        } else {
-            $output['config'] = null;
-        }*/
```

Comment:
@KodeStar I removed this, I don't think it was used, but what do you think?

Reply:
Yeah, no problem